x-pack/plugin/sql/jdbc/qa/build.gradle

```diff
@@ -0,0 +1,66 @@
+description = 'Integration tests for SQL JDBC'
+apply plugin: 'elasticsearch.build'
+archivesBaseName = 'qa-sql-jdbc'
+group = "org.elasticsearch.x-pack.qa.sql.jdbc"
```

Comment:
We shouldn't need to override group or `archivesBaseName` since we aren't going to publish this module.